<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioWidget</class>
 <widget class="QWidget" name="AudioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_44">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addPushButton">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removePushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Audio properties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="audioNameLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Audio name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pictureFrameRateLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Picture framerate:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="pictureFrameRateSpinBox">
         <property name="maximum">
          <double>16777215.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="languageLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="languageLineEdit">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QGroupBox" name="channelsGroupBox">
         <property name="title">
          <string>Chanels</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="channelsTableWidget">
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Path</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="audioNameLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>channelsTableWidget</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>AudioWidget</receiver>
   <slot>refreshAudio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pictureFrameRateSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>AudioWidget</receiver>
   <slot>refreshAudio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AudioWidget</receiver>
   <slot>refreshAudio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioNameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AudioWidget</receiver>
   <slot>refreshAudio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
